migrations/20200425013627-change-tags-to-lowercase.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.sequelize.query(
+      `
+        UPDATE ONLY "Conversations" c
+        SET "tags" = c_with_lowercase_tags."tags"
+        FROM (SELECT id, lower(tags::text)::text[] as tags from "Conversations") as c_with_lowercase_tags
+        WHERE c.id = c_with_lowercase_tags.id AND c."tags" IS NOT NULL
```

Comment:
Are these join (`FROM`) really necessary? Your approach is good, but I think we can simplify it: 

```suggestion
        UPDATE ONLY "Conversations" c
        SET "tags" = lower(tags::text)::text[] 
        WHERE "tags" IS NOT NULL
```